contracts/controller/src/healthfactor.rs

```diff
@@ -6,7 +6,7 @@ impl Contract {
         if let Some(map) = map_raw {
             for (asset, balance) in map.iter() {
                 let price = self.get_price(asset).unwrap();
-                result += Percentage::from(price.volatility).apply_to(price.value * balance);
+                result += Percentage::from(Percent::from(price.volatility)).apply_to(Balance::from(price.value) * balance);
```

Comment:
@dariacherniak why do we need nested `Percentage::from(Percent::from(price.volatility))` ?